example-cropping/README.md

```diff
@@ -0,0 +1,779 @@
+        >
+         Upload
+        </Button>
+     
+
+    </div>
+  )
+
+};
+
+export { UploadImage };
+```
+
+example usage
+
+```js
+  <UploadImage appwrite={appwrite} />
+```
+
+this only take one props which is appwrite for connecting to our server using appwrite sdk also if you notice this code
+
+```js
+await props.appwrite.storage.createFile(uploadFile, ['*'], ['*']);
+```
+
+is actually for uploading our file whether its image or but in this tutorial we will use image and this function actually take three parameters you can take a look the detailed explanation here https://appwrite.io/docs/client/storage#createFile
+
+## Step 5 - Create ListImage Component
+
+So after we done uploading our image we want to see all of our image that we upload right? so in order to do that we create a ```ListImage``` component so that we can show our image from the server
+
+```js
+import React, { useState, useEffect } from 'react';
+import { PreviewImage } from "./PreviewImage"
+import Button from "@material-ui/core/Button";
+import Grid from "@material-ui/core/Grid";
+function ListImage(props) {
+  const [listImages,setListImages]=useState([])
+  const [active, setActive] = useState(null)
+  async function getAllImages() {
+    let images = await props.appwrite.storage.listFiles();
+    let me=[...images.files];
+   
+    setListImages(me)
+
+  }
+
+  useEffect(() => {
+    getAllImages()
+
+  }, []);
+  useEffect(() => {
+
+
+  }, [active]);
+ 
+  async function selectImage(e, id, index) {
+
+    setActive(index)
+    props.changeImage(id)
+
+  }
+  return (
+    <div>
+      <h1>List Image</h1>
+      
+<Button
+          style={{marginBottom:30}}
+          variant="contained"
+          color="primary"
+          onClick={()=>getAllImages()}
+        >
+      Refresh List
+        </Button>
+  
+      
+        <Grid
+  container
+  direction="row"
+  spacing={5}
+>
+{listImages?.length && listImages.length > 0 ? listImages.map((a, index) => {
+        a.color = active === index ? "solid" : "none";
+
+        return (
+          <Grid container justify="center" xs={4} item key={index} style={{  border: a.color, cursor: "pointer" }} onClick={(e) => selectImage(e, a.$id, index)}>
+
+            <PreviewImage width={300} output={"webp"} appwrite={props.appwrite} id={a.$id} />
+          </Grid>
+
+        )
+      }) : ""}
+</Grid>
+      
+    </div>
+  )
+
+};
+
+export { ListImage };
+
+```
+example usage
+
+```js
+<ListImage appwrite={appwrite} changeImage={(id) => changeImage(id)} />
+```
+
+there is two props here:
+
+- ```appwrite``` for using the appwrite SDK
+- ```changeImage``` to pick or change an image that we want to crop later in ```PreviewAndCrop``` component
+
+## Step 6 - Create PreviewAndCrop Component
+
+So what this component does is it takes the imake that we pick from the list images then we change the value of the width, height etc. here. So that we can preview how the image look like after cropping.
```

Comment:
```suggestion
So what this component does is it takes the image that we pick from the list of images and allows us to change the value of the width, height etc. here. So that we can preview how the image looks after cropping.
```